runtime/common/src/paras_registrar.rs

```diff
@@ -271,31 +274,52 @@ pub mod pallet {
 		pub fn swap(origin: OriginFor<T>, id: ParaId, other: ParaId) -> DispatchResult {
 			Self::ensure_root_para_or_owner(origin, id)?;
```

Comment:
This will basically need to be called by Root unless we write specific logic around the owner of the same parachain id being able to do a swap even when the parachain is "locked".